docs/content/manual/manual.yml

```diff
@@ -1109,6 +1109,19 @@ sections:
             input: '["foo", "bar", "baz"]'
             output: ['["foo"]']
 
+      - title: "`compact`"
+        body: |
+
+          The builtin function `compact` removes nulls from an object.
```

Comment:
from object and array? maybe mention that it is recursive?

Reply:
Addressed in https://github.com/jqlang/jq/pull/2838/commits/7052fa1d867aceacac529248c4165be24823361d